### ISSUE TITLE
Assembly Viewer doesn't use any localization text

### ISSUE THREAD

**Issue by smallg0at** [logisim-evolution/logisim-evolution#1748]
The Assembly Viewer currently uses hardcoded English text for its UI, so localization efforts are unable to do any contribution to this module.

**Comment by BFH-ktt1** [comment]
@smallg0at : This is correct, unfortunately there are some more parts in logisim that still are not able to be localized and have hardcoded English text. This is still on the TODO list.
